forloop_modules/utils/definitions.py

```diff
@@ -135,3 +138,7 @@
 JSON_SERIALIZABLE_TYPES_AS_STRINGS = [typ.__name__ for typ in JSON_SERIALIZABLE_TYPES]
 REDIS_STORED_TYPES = [pd.DataFrame, datetime.datetime]
 REDIS_STORED_TYPES_AS_STRINGS = [typ.__name__ for typ in REDIS_STORED_TYPES] + ["function", "class"]
+
+# Regex used for validating URLs across the whole app
+# NOTE: Regex created by Joaquin, simply mirrored on backend
```

Comment:
I would be happier to read here in the comment what this regex does exactly so I dont need to decrypt it :)